crates/ethportal-api/src/discv5.rs

```diff
@@ -40,4 +37,21 @@ pub trait Discv5Api {
     /// Fetch the ENR representation associated with the given Node ID.
     #[method(name = "lookupEnr")]
     async fn lookup_enr(&self, node_id: NodeId) -> RpcResult<Enr>;
+
+    /// Look up ENRs closest to the given target
+    #[method(name = "recursiveFindNodes")]
+    async fn recursive_find_nodes(&self, node_id: NodeId) -> RpcResult<Vec<Enr>>;
+
+    /// Send a TALKREQ request with a payload to a given peer and wait for response.
+    #[method(name = "talkReq")]
+    async fn talk_req(&self, enr: Enr, protocol: String, request: Vec<u8>) -> RpcResult<Bytes>;
```

Comment:
```suggestion
    async fn talk_req(&self, enr: Enr, protocol: Subnetwork, request: Vec<u8>) -> RpcResult<Bytes>;
```

Couldn't we just use Subnetwork here?